### ISSUE TITLE
Empty strings in the group cause paru to fail

### ISSUE THREAD

**Issue by nachtjasmin** [Morganamilo/paru#988]
### Affected Version
**paru -V** paru v1.11.1 - libalpm v13.0.2

### Description
**Have you checked previous issues?** yes

It looks like `paru` has a problem with the empty groups of the [mechvibes AUR package](https://aur.archlinux.org/packages/mechvibes). Cloning the PKGBUILD and running `makepkg -Si` works fine tho.

### Output
**Include the FULL output of any relevant commands/configs**

```shell
$ paru -S mechvibes             
:: Löse Abhängigkeiten auf…
:: Berechne Konflikte…
:: Berechne interne Konflikte…

Aur (1) mechvibes-2.3.0-1

:: Weiter überprüfen? [J/n]: j

:: Lade PKGBUILDs herunter…
 (1/1) mechvibes-2.3.0-1                             [------------------------------------------------------]
Fehler: kann .SRCINFO für ‚mechvibes-2.3.0-1‘ nicht parsen: key 'groups' requires a value: Line 8: groups =
```

**Comment by marianomarciello** [comment]
HI @chloe-the-catgirl I searched a bit about this issue, and it seems to me that this is related to the `srcinfo` create [https://github.com/Morganamilo/srcinfo.rs](link). Seems like the groups field must have some value if present. This makes sense to me because the groups field in a PKGBUILD file contains info about the group that this package belongs to. (According to the arch doc [link](https://wiki.archlinux.org/title/PKGBUILD#groups)). So an empty groups what does it mean ? Why just don't remove the groups from the PKGBUILD file ?

**Comment by chloe-the-catgirl** [comment]
I didn't create this package and although I could remove the empty groups easily with paru, I got confused first. yay is building the package just fine, therefore I've created this issue. Maybe the maintainers want compatibility with yay, maybe they care about more about accuracy.

This issue was created to inform the maintainers about the build errors and the behaviour of other AUR helpers. It's up to them how they wanna handle this.
